modules/admin_manual/pages/configuration/user/user_management.adoc

```diff
@@ -136,79 +114,46 @@ There are 4 types of quota settings in ownCloud when dealing with LDAP users.
 
 === Quota Field
 
-Found in menu:User Authentication[the Advanced Tab > Special Attributes],
-this setting overwrites the rest. If set, this is what will be set for an LDAP user’s quota in ownCloud.
+Found in menu:User Authentication[the Advanced Tab > Special Attributes], this setting overwrites the rest. If set, this is what will be set for an LDAP user’s quota in ownCloud.
 
 === Quota Default
 
-Found in menu:User Authentication[the Advanced Tab > Special Attributes], this is the fallback option
-if no quota field is defined.
+Found in menu:User Authentication[the Advanced Tab > Special Attributes], this is the fallback option if no quota field is defined.
 
 === User Quota
 
 This is what you set in the web UI drop down menu, and is how you set user quota.
 
 === Default Quota
 
-This will be set if no quota is set, and is found in menu:Users Tab[Gear Wheel > Default Quota].
-If *Quota Field* is not set, but *Quota Default* is, and a systems administrator tries to set
-a quota for an LDAP user with *User Quota*, it will not work, since it is overridden by
-*Quota Default*.
-
-Click the btn:[gear] icon on the lower left pane to set a default storage quota.
-This is automatically applied to new users. You may assign a different
-quota to any user by selecting from the *Quota* dropdown, selecting
-either a preset value or entering a custom value. When you create custom
-quotas, use the normal abbreviations for your storage values such as 500
-MB, 5 GB, 5 TB, and so on.
+This will be set if no quota is set, and is found in menu:Users Tab[Gear Wheel > Default Quota]. If the *Quota Field* is not set, but *Quota Default* is, and a systems administrator tries to set a quota for an LDAP user with *User Quota*, it will not work, since it is overridden by *Quota Default*.
```

Comment:
```suggestion
This will be used if no quota is set and is found in menu:Users Tab[Gear Wheel > Default Quota]. If the *Quota* is not set, but *Default Quota* is, and a systems administrator tries to set a quota for an LDAP user with *User Quota*, it will not work since it is overridden by *Default Quota*.
```